### DISCUSSION TITLE
Should `org-modern-symbol` inherit the font from `default`?

### DISCUSSION THREAD

**Discussion by elken** [minad/org-modern#254] [Q&A]
Title says it all really, apologies if this is categorised incorrectly or raised before (I did search first).

Users like myself of variable-pitch-mode observe behaviour similar to the attached (not my video, but I also get similar with org-modern).

https://github.com/user-attachments/assets/88fca346-3007-450c-b3b5-bc73fe30e343

A simple fix is to set the `org-modern-symbol` face specifically to use the same font family as default, but I think this is a sane default in general. Regular users should notice no difference, and variable/mixed-pitch users get the benefit of having better proportioned symbol characters.

WDYT minad?

**Comment by minad** [comment]
It is not common for faces to inherit from `default`, so my tentative answer is no. It is not clear to me why `org-modern-symbol` should be treated any differently than other faces.

**Reply by elken** [reply]
That's fair then, so the reasoning is in variable/mixed-pitch-mode the headings will inherit the variable pitch font and as you see in the video, inhibits mis-proportioned text that often doesn't align right.

You're also right that it maybe shouldn't be _just_ that face, any face that is likely to end up in a heading really.

**Reply by minad** [reply]
I don't see the video since my setup is a little borked right now after my switch to qutebrowser. My familarily with `variable-pitch-mode` is limited. But as far as I understand it needs a lot of tweaking of many elements which are unhappy with variable pitch. That being said in my setup `variable-pitch-mode` looks pretty good with `org-modern-mode`, except for the tables obviously. So maybe you could show me a screenshot of the problem you are facing?

**Reply by jakkrz** [reply]
Hey, I'm the original creator of that video. Here are some screenshots of what I believe to be the issue discussed with org-modern (using a fixed-pitch font and having org-indent-mode enabled).
![image](https://github.com/user-attachments/assets/a5419a60-77a8-443f-90e5-921317f3c63a)
![image](https://github.com/user-attachments/assets/16e3aaec-c58a-42d1-b0c9-566d51f8b0bd)
There's no reason things should be aligned correctly when zoomed out vs. zoomed in.

**Reply by minad** [reply]
Oh I see, org-modern was never supposed to preserve aligment. The bullets and the TODO/priority/tag formatting did always break alignment. See for example the old screenshot in the org-modern README. The alignment in your first screenshot is just luck.

**Reply by elken** [reply]
Okay no I wasn't talking about alignment at all, that's a whole other unrelated thing taken care of elsewhere.

> My familarily with variable-pitch-mode is limited

No problem, basically it just uses a non-monospace font for the headings and other textual elements that are meant to be prose, and keeps the default monospaced (ideally, but obviously not mandatory) font reserved for code blocks and ~~ == items. The issue it creates here is that, as the symbols are part of the heading, they inherit the same font and thus stop being nicely monospaced symbols that fit.

> So maybe you could show me a screenshot of the problem you are facing

This is what it looks like by default
![image](https://github.com/user-attachments/assets/f3b8f401-12b7-4ee6-8ffd-39a65899b619)

And with my change to set `org-modern-symbol` to my default monospace font
![image](https://github.com/user-attachments/assets/d8b1fd5b-5619-4852-a01c-3cd211363bb9)

You see that it's now neatly lined up in the middle.

As I say, you're right that `org-modern-symbol` isn't the only face that should get this default, probably all of them should, but I'll leave the call up to you; just bringing this issue to your attention 😄
